### PR TITLE
Patched welcome null msg issue

### LINKED ISSUE #897: Username is shown as "null" on login
**Summary:**

Username is shown as "null" on login

**Steps to reproduce:**

Login to app

**Expected behaviour:**

Username should be shown

**Observed behaviour:**

Username is shown as null

**Device and Android version:**

Xiaomi, Marshmallow

**Screenshots:**

### PR DESCRIPTION
**Pull request by etlhsu**
Fixes #897

image

Now, the app receives the username properly, from the same Retrofit API call. My solution requires no refactoring and no corner-cutting. The solution best fits the needs of the issue, with the rest of the product untouched.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -134,5 +134,8 @@ dependencies {
     androidTestImplementation "com.android.support.test.espresso:espresso-core:$rootProject.espressoVersion"
     androidTestImplementation "com.android.support.test:runner:$rootProject.runnerVersion"
     androidTestImplementation "com.android.support.test:rules:$rootProject.rulesVersion"
+
+    //Multidex
```
Comment (miPlodder) [inline app/build.gradle:138]: @ethan627hsu Create a separate PR for this as its not relevant for this issue and undo these changes.

File: app/src/main/java/org/mifos/mobilebanking/models/User.kt [hunk 2]
```diff
@@ -8,9 +9,9 @@ import java.util.ArrayList
  */
 
 data class User (
-
     var userId: Long = 0,
     var isAuthenticated: Boolean = false,
+    @SerializedName(value = "username")
```
Comment (miPlodder) [inline app/src/main/java/org/mifos/mobilebanking/models/User.kt:14]: @ethan627hsu You approach is also correct, but you change the _userName_ to _username_.

File: build.gradle [hunk 3]
```diff
@@ -19,8 +19,8 @@ buildscript {
 
 allprojects {
     repositories {
-        jcenter()
         google()
+        jcenter()
```
Comment (miPlodder) [inline build.gradle:23]: @ethan627hsu Create a separate PR for this as it's not relevant for this issue and undo these changes here.

**Comment by miPlodder** [comment]
@ethan627hsu Also, follow the [commit guidelines](https://github.com/openMF/mifos-mobile/wiki/Commit-Style-Guide).

**Comment by etlhsu** [comment]
I will decouple my requested changes...
